openhcl/virt_mshv_vtl/src/lib.rs

```diff
@@ -275,7 +272,8 @@ impl From<EnterMode> for hcl::protocol::EnterMode {
 
 #[cfg(guest_arch = "x86_64")]
 #[derive(Inspect)]
-struct UhCvmPartitionState {
+/// Partition-wide state for CVMs.
+pub struct UhCvmPartitionState {
```

Comment:
Why is this pub?

Reply:
private in public issues. it's on a trait used in bounds on methods on UhProcessor, which is pub.